src/App.tsx

```diff
@@ -9,10 +15,19 @@ function App(): any {
   };
 
   return (
-    <div className="">
-      <Header handleSubmit={handleSubmit} />
-      <div className="mainBlock">{searchEntry}</div>
-    </div>
+    <>
+      <ErrorBoundary>
+        <Header handleSubmit={handleSubmit} />
+      </ErrorBoundary>
+      <Layout>
+        <ErrorBoundary message={DATA_LOADING_ERROR}>
+          <MoviesList searchEntry={searchEntry} />
+        </ErrorBoundary>
+      </Layout>
+      <Footer>
+        <FooterEmail></FooterEmail>
```

Comment:
Since this element has no children, it can be written as `<FooterEmail />` instead of `<FooterEmail></FooterEmail>`

Reply:
Completely agree, will fix it